utils/include/detray/detectors/create_toy_geometry.hpp

```diff
@@ -38,6 +38,52 @@ using point3 = __plugin::point3<detray::scalar>;
 using vector3 = __plugin::vector3<detray::scalar>;
 using point2 = __plugin::point2<detray::scalar>;
 
+/// Configure the toy detector
+struct toy_det_config {
+    /// No. of barrel layers the detector should be built with
+    unsigned int m_n_brl_layers{4u};
+    /// No. of endcap layers (on either side) the detector should be built with
+    unsigned int m_n_edc_layers{3u};
+    /// Field vector for an homogenoues b-field
+    vector3 m_bfield_vec{0.f, 0.f, 2.f * unit<scalar>::T};
+    /// Input file name for the covfie b-field
+    std::string m_bfield_file{!std::getenv("DETRAY_BFIELD_FILE")
+                                  ? ""
```

Comment:
Will this give the user a satisfying error? It's probably good to make sure that the user knows they should set this environment variable.

Reply:
I think we only use this variable in the CI. User will actually have to pass a filename to the reader config